wandb/sdk/artifacts/artifact.py

```diff
@@ -2102,6 +2160,60 @@ def _load_manifest(self, url: str) -> None:
                 dep_artifact = entry._get_referenced_artifact(self._client)
                 self._dependent_artifacts.add(dep_artifact)
 
+    @staticmethod
+    def _get_gql_artifact_fragment() -> str:
+        fields = InternalApi().server_create_artifact_introspection()
+        fragment = """
+            fragment ArtifactFragment on Artifact {
+                id
+                artifactSequence {
+                    project {
+                        entityName
+                        name
+                    }
+                    name
+                }
+                versionIndex
+                artifactType {
+                    name
+                }
+                description
+                metadata
+                ttlDurationSeconds
+                aliases {
+                    artifactCollection {
+                        project {
+                            entityName
+                            name
+                        }
+                        name
+                    }
+                    alias
+                }
+                state
+                commitHash
+                fileCount
+                createdAt
+                updatedAt
+            }
+        """
+        if "ttlDurationSeconds" not in fields:
+            return fragment.replace("ttlDurationSeconds", "")
+        return fragment
+
+    @staticmethod
```

Comment:
Nit: why is this static? You're passing in the attributes of an instance, why not just make it an instance method?